references/dump.md

```diff
@@ -0,0 +1,46 @@
+# Dump
+
+The `dump` route allow the creation of database dumps. Dumps are compatible between MeiliSearch versions.
```

Comment:
```suggestion
The `dumps` route allow the creation of database dumps. Dumps are compatible between MeiliSearch versions.
```

Reply:
Seems like a good suggestion ! Could you change the typo here : 
`the process will take som etime`
Its `the process will take some time` :)